.github/workflows/project-board.yml

```diff
@@ -0,0 +1,16 @@
+# This internal workflow track the status of PRs opened by the core team.
+# Open-source contributors should not worry about this workflow.
+name: 'Project Board'
+on:
+  pull_request:
```

Comment:
The way I interpret the combination of this and `github.triggering_actor` below, this means that any time we take any action on a PR, including adding/removing labels, editing PR descriptions, closing/reopening, or [anything else listed here in the docs](https://github.com/actions/add-to-project?tab=readme-ov-file#creating-a-pat-and-adding-it-to-your-repository), this workflow will run an forcefully add something to the project board. This seems pretty disruptive. Can we limit this to just `opened` events?

Reply:
Yes, that sounds good. This was intended to only run once.